templates/base_layout.html

```diff
@@ -80,24 +80,25 @@
   <link rel="stylesheet" type="text/css" media="screen" href="{{ versioned_static('css/styles.css') }}">
 </head>
 
-<body class="no-js is-paper l-site">
+
```

Comment:
This is a bit of refactoring that is required to make base layout support standard site layout and new docs layout.